src/login/logind-core.c

```diff
@@ -380,13 +381,42 @@ bool manager_shall_kill(Manager *m, const char *user) {
         return m->kill_user_processes;
 }
 
+int config_parse_n_autovts(const char *unit, const char *filename, unsigned line, const char *section, unsigned section_line, const char *lvalue, int ltype, const char *rvalue, void *data, void *userdata)
+{
```

Comment:
nitpick: opening brackets for functions please on the same line as the function name. Please see CODING_STYLE